sql/moz-fx-data-shared-prod/accounts_backend_derived/users_services_daily_v1/checks.sql

```diff
@@ -48,6 +48,80 @@ check_results AS (
     )
 )
 SELECT
-  IF(COUNT(*) > 0, ERROR('Events count mismatch between accounts_events and events pings'), NULL)
+  IF(
+    COUNT(*) > 0,
+    ERROR('Events count mismatch between backend accounts_events and events pings'),
+    NULL
+  )
+FROM
+  check_results;
+
+#warn
+-- check if we're receiving the same number of events in frontend `accounts_events` and `events` pings
+WITH events_new AS (
+  SELECT
+    DATE(e.submission_timestamp) AS day,
+    CONCAT(event.category, "_", event.name) AS event_name,
+    COUNT(*) AS count_new
+  FROM
+    `moz-fx-data-shared-prod.accounts_frontend_stable.events_v1` AS e
```

Comment:
We don't have any derived tables scheduled in bw-etl for `accounts_frontend` so I'm adding this check here together with `accounts_backend`.